modular_skyrat/modules/pet_owner/pet_owner.dm

```diff
@@ -28,6 +28,12 @@
 
 	var/obj/item/pet_carrier/carrier = new /obj/item/pet_carrier(get_turf(quirk_holder))
 	var/mob/living/simple_animal/pet/pet = new pet_type(carrier)
+	var/new_name = client_source?.prefs.read_preference(/datum/preference/text/pet_name)
+	if (new_name)
+		pet.name = new_name
+	var/new_desc = client_source?.prefs.read_preference(/datum/preference/text/pet_desc)
+	if (new_desc)
```

Comment:
You might want to trim whitespaces from the start and end of the names and descriptions, to also make sure that they're not just whitespace, so we don't have people naming their pet ` `, y'know. It'll look nicer, and should mean that the check will work as long as they have actual characters in it.

Reply:
regex :)